service/src/main/java/uk/gov/hmcts/reform/fpl/service/RepresentativeService.java

```diff
@@ -211,4 +214,29 @@ private void associatedRepresentativeWithParty(CaseData caseData, Element<Repres
                 return Optional.empty();
         }
     }
+
+    public List<Element<Representative>> getRepresentativePartiesToNotify(CallbackRequest request) {
```

Comment:
Seems like a lot to send for just using the representatives. Thoughts on changing it so that it just accepts 2 lists of representatives, one from before and one for current?

Reply:
really like this idea :)